src/Components/App/App.js

```diff
@@ -4,13 +4,15 @@ import Movies from '../Movies/Movies'
 import Header from '../Header/Header'
 import Footer from '../Footer/Footer'
 import MovieDetails from '../MovieDetails/MovieDetails'
+import ReRoute from '../ReRoute/ReRoute'
 import { getAllMovies, getSingleMovie, getSingleMovieVideo } from '../../util'
+import { Route, Switch } from 'react-router-dom'
 
 class App extends Component {
   constructor() {
     super()
     this.state = { 
-      movies: [],
+      movies: {movies: []},
```

Comment:
Did we decide to keep this? Or should we add a conditional for a loading screen if movies is an empty array at first?